ytfzf

```diff
@@ -126,6 +126,31 @@ dep_ck "jq" "youtube-dl";
 #don't check $YTFZF_PLAYER as it could be multiple commands
 [ "$YTFZF_PLAYER" = "mpv" ] && dep_ck "mpv"
 
+#create subscriptions file from data downloaded from youtube
+create_subs () {
+
+    FILE="${config_dir}/subscriptions.json"
+    
+    # there is a decent guide on getting this data by the FreeTube devs   
+    if [ ! -f "$FILE" ]; then
+        printf "\033[31mYou need to have a subscriptions.json file in the ${config_dir} directory!\033[0m\n"
+        printf "For further information visit https://github.com/FreeTubeApp/FreeTube/wiki/Importing-Your-YouTube-Subscriptions !\n"
+        exit
+    fi
+   
+    # start fresh
+    > $config_dir/subscriptions
+
+    # check how many subscriptions there are in the file
+    sublength=$(cat "$FILE" | jq '. | length')
+
+    for i in $( seq 0 $((sublength-1)) ); do
+
+        channelId=$(cat subscriptions.json | jq --argjson index ${i} '.[$index] .snippet .resourceId .channelId' )
+        echo "https://www.youtube.com/channel/${channelId:1:24}/videos" >> "$subscriptions_file"
```

Comment:
I don't think you could do this in posix, we could  maybe you could do something like `${channelId%${channelId#??????????????????????}}`

Reply:
That didn't do the trick for me, the first double quote wasn't cut off. I piped the variable into 'cut' : 
`$(echo ${channelId} | cut -c 2-25)`